configobject/history/history.go

```diff
@@ -582,3 +605,11 @@ func removeEntryFromEntriesSlice(s []redis.XMessage, i int) []redis.XMessage {
 	s[len(s)-1], s[i] = s[i], s[len(s)-1]
 	return s[:len(s)-1]
 }
+
+func defaultIfNil(value interface{}, defaultValue interface{}) interface{} {
```

Comment:
* Docs would be helpful
* You don't need to specify the same param type twice in a row